src/InlineHtmlStripStylesTransformer.ts

```diff
@@ -125,65 +86,55 @@ export function factory(cs: ConfigSet) {
    * Traverses the AST down to the relevant assignments in the decorator
    * argument and returns them in an array.
    */
-  function getPropertyAssignmentsToTransform(classDeclaration: ClassDeclaration) {
-    return flatten<PropertyAssignment>(classDeclaration.decorators!
-      .map(dec => dec.expression)
-      .filter(ts.isCallExpression)
-      .map(callExpr => (callExpr as CallExpression).arguments
-        .filter(ts.isObjectLiteralExpression)
-        .map(arg => (arg as ObjectLiteralExpression).properties
-          .filter(ts.isPropertyAssignment)
-          .map(arg => arg as PropertyAssignment)
-          .filter(propAss => ts.isIdentifier(propAss.name))
-          .filter(propAss => TRANSFORM_PROPS.includes((propAss.name as Identifier).text))
-        )
-      )
-    )
+  function isPropertyAssignmentToTransform(node: Node): node is PropertyAssignment {
+    return ts.isPropertyAssignment(node) &&
+      ts.isIdentifier(node.name) &&
+      TRANSFORM_PROPS.includes(node.name.text)
   }
 
   /**
    * Clones the node, identifies the properties to transform in the decorator and modifies them.
    * @param node class declaration with decorators
    */
```

Comment:
ts docs are outdated - it is not about classes anymore, just about an assignment